jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -799,7 +800,8 @@ private InterpreterResult executeSql(String sql,
           // so we need to trim it first in this case.
           sqlToExecute = sqlToExecute.trim();
         }
-        LOGGER.info("Execute sql: " + sqlToExecute);
+        LOGGER.info(String.format("[%s|%s|%s] Execute sql: %s",
+                user, noteId, paragraphId, sqlToExecute));
```

Comment:
```suggestion
        LOGGER.info("[{}|{}|{}] Execute sql: {}", user, noteId, paragraphId, sqlToExecute);
```
See: https://rules.sonarsource.com/java/RSPEC-2629/

Reply:
Yes, I also think it's better.